discordHelper.js

```diff
@@ -0,0 +1,17 @@
+import fetch from "node-fetch";
+
+export async function checkEventIsNovel (event) {
+  const messages = await getAllMessages()
+  
+  return !messages.some(message => {
+      return message.embeds.some(embed => embed.url === event.shortUrl)
+    })
+}
+
+async function getAllMessages () {
+  const res = await fetch(`https://discord.com/api/channels/${process.env.ANNOUNCEMENT_CHANNEL_ID}/messages`, {
```

Comment:
Is there a way to fetch messages from discord via the library?

Reply:
I don't think we should move forward with this approach.

Let's suppose this API only returns the last 10 messages. What if someone writes ten messages after this call, and then the previous event is no longer in the history? It'll post again. The other issue is that we can't return all messages either to check against.